test/tall/regression/1600/1628.unit

```diff
@@ -0,0 +1,10 @@
+>>>
+bool f(bool b, int i) {
+  return // Check b.
+  b == true || i.isEven;
```

Comment:
This reproduction is missing the `// Check i.` comment from the failure case in the filed issue

Reply:
Oh dear, good catch and poor reading comprehension on my part. Added. (Fortunately, that case is passing too.)